data/card_catalogue/anger.json

```diff
@@ -15,41 +18,50 @@
     "tier": "green",
     "details": {
       "rules": {
-	      "active": "A visible target is forced to stop what they're doing and discuss their feelings with you",
-	      "passive": "Whenever you make an attack, you can choose whether or not it will deal damage"
+	      "active": "A visible character is forced to stop what they're doing and discuss their feelings with you.",
+	      "passive": "Whenever you make an attack, you can choose whether or not it will deal damage."
       },
       "flavour": "\"I guess, in a way, I've just been trying to impress my old man this whole time\" - Brutus the Bloody Impaler",
-      "image": "assets/core/card/illustration/Lunge.png"
+      "image": "assets/core/card/illustration/Talk it Out.png"
     }
   },
   {
     "title": "Timeout",
     "upgrade": "Change of Heart",
     "tier": "red",
     "details": {
-      "rules": "For one minute, non-player characters will not become hostile unless they receive damage",
-      "flavour": "\"There goes Swift Sammy, taking money out of our vault again. What a great guy.\" - Regis, soon to be unemployed security guard",
-      "image": "assets/core/card/illustration/Lunge.png"
+      "rules": {
+        "active": "For one minute, NPCs will not become hostile unless they take damage.",
+        "passive": ""
```

Comment:
Nit: We should use null instead of an implicit null interpretation of an empty string

Reply:
Tracked here: https://github.com/elliottomlinson/rpcg/issues/114